lib/engine.py

```diff
@@ -583,7 +589,11 @@ def format_insert_value(self, value, datatype, escape=True):
                     strvalue = self.escape_double_quotes(strvalue)
                 if hasattr(self.table, "escape_single_quotes") and self.table.escape_single_quotes:
                     strvalue = self.escape_single_quotes(strvalue)
-            return "'" + strvalue + "'"
+                return "'" + strvalue + "'"
+            if processed:
+                return strvalue
+            else:
+                return "'" + strvalue + "'"
```

Comment:
The new `processed` code needs an explanation in the docstring. It's not intuitive to me what's being accomplished here.